scio-bigquery/src/main/scala/com/spotify/scio/bigquery/client/TableOps.scala

```diff
@@ -167,6 +162,13 @@ final private[client] class TableOps(client: Client) {
     client.underlying.tables().get(p, tableRef.getDatasetId, tableRef.getTableId).execute()
   }
 
+  /** Get list of tables in a dataset. */
+  def tableReferences(tableRef: TableReference): Seq[TableReference] =
```

Comment:
The input `TableReference` can have `tableId`? That can be a little misleading? It's unclear from the call site that `.tableReferences(tableRef)` gets all tables from the dataset `tableRef` is in.

Reply:
yeah that's true... agree it might be confusing